src/dropdown/Dropdown.js

```diff
@@ -115,10 +133,85 @@ export default class Dropdown extends React.Component {
 
   focus = () => { this.select.focus(); }
 
+  optionGroupRenderer = ( title ) => {
+    if (title === 'undefined') {
+      return null;
+    }
+    else {
+      return (
+        <FlexView className='Select-option-group'>
+          {this.props.optionGroupRenderer(title)}
+        </FlexView>
+      );
+    }
+  }
+
+  _menuRenderer = ({
```

Comment:
I'd call this `defaultMenuRenderer` and I'll pass it as defaultProp exactly in the same way you're doing for `optionGroupRenderer`.
Also, you're completely ignoring the `menuRenderer` coming from props.